molecule_libvirt/playbooks/create.yml

```diff
@@ -0,0 +1,259 @@
+---
+
+- name: Create
+  hosts: localhost
+  gather_facts: false
+  no_log: "{{ molecule_no_log }}"
+  vars:
```

Comment:
Could be nice to add a new line between `no_log` and `vars`.

Reply:
Done !

Reply:
done.

Reply:
ok.